src/test/java/org/fundacionjala/at15/katas/calculator/CalculatorTest.java

```diff
@@ -9,14 +9,24 @@
     @Test
     public void itShouldSumTwoIntegers() {
         Calculator calculator = new Calculator();
-        int result = calculator.doOperation(new SumOperation(), 2, 3);
+        int[] operands = {2, 3};
+        int result = calculator.doOperation(new SumSubstractOperation(), operands);
         assertEquals(5, result);
     }
-
+    
     @Test
     public void itShouldSubstractTwoIntegers() {
         Calculator calculator = new Calculator();
-        int result = calculator.doOperation(new SubstractOperation(), 5, 1);
+        int[] operands = {5, -1};
+        int result = calculator.doOperation(new SumSubstractOperation(), operands);
         assertEquals(4, result);
     }
+    
+    @Test
+    public void itShouldSumAndSubstractManyArguments() {
+        Calculator calculator = new Calculator();
+        int[] operands = {2, -3, 5, 1, -2};
```

Comment:
I have to create a functionality that allows me to create this array to be able to add and subtract several arguments at the same time